meeshkan/serve/mock/storage.py

```diff
@@ -1,28 +1,106 @@
 import logging
+import uuid
+
+from http_types import Request
+from openapi_typed import OpenAPIObject
 
 logger = logging.getLogger(__name__)
 
 
-class StorageManager:
+class Storage:
     def __init__(self):
         self._default = dict()
-        self._storages = dict()
+        self._entities = dict()
 
     @property
     def default(self):
         return self._default
 
-    def get(self, name):
-        return self._storages.get(name)
+    def add_entity(self, name):
+        self._entities[name] = dict()
+
+    def clear(self):
+        self._default.clear()
+        for entity in self._entities.values():
+            entity.clear()
```

Comment:
What's the type definition for an `Entity`? Apparently it has a `clear` method?

Reply:
Please, revisit entities logic. It changed significantly.